client/src/components/Menu/index.tsx

```diff
@@ -181,8 +185,11 @@ class Menu extends React.Component<IMenuProps> {
                                             ))}
                                         </Link>
                                     ),
-                                ) : <span className='ml-3'>No results...</span>}
-                                {!!searchResults.length && !loaded &&
+                                ) : !loading && <span className='ml-3'>No results...</span>}
+                                <div className='d-flex justify-content-center'>
+                                    {loading && <Spinner color='dark'/>}
+                                </div>
+                                {searchResults.length % RESULTS_PER_PAGE === 0 && !loaded && !loading &&
```

Comment:
const RESULTS_PER_PAGE = 5; never bacome RESULTS_PER_PAGE === 0 ?

Reply:
I used modulo operator (%) here. So, searchResults.length % RESULTS_PER_PAGE can be 0.

But it actually shouldn't be here, because of next condition. Fixed and answered in other comment.